### PR TITLE
add splash page

### PR DESCRIPTION
**Pull request by devkishor8007**
adding the splash page

### CODE REVIEW

File: lib/login.dart [hunk 1]
```diff
@@ -51,9 +51,9 @@ class _LoginState extends State<Login> {
                             elevation: 5.0,
                           ),
                           onPressed: () {
-                            if (emailController.text == 'anant1234@gmail.com' &&
+                            if (emailController.text == 'devkishor@gmail.com' &&
```
Comment (anantcodes) [inline lib/login.dart:54]: static email must not be changed.

File: lib/login.dart [hunk 2]
```diff
@@ -51,9 +51,9 @@ class _LoginState extends State<Login> {
...
                                 passwordController.text ==
-                                    'anant') // email and password static
+                                    'devkishor') // email and password static
```
Comment (anantcodes) [inline lib/login.dart:56]: static password must not be changed.

File: lib/login.dart [hunk 3]
```diff
@@ -63,6 +63,8 @@ class _LoginState extends State<Login> {
                                 backgroundColor: Colors.grey,
                                 gravity: ToastGravity.BOTTOM,
                               );
+                              emailController.clear();
```
Comment (anantcodes) [inline lib/login.dart:66]: Do not clear the credentials as of now. This functionality will be adapted later.

**Comment by devkishor8007** [comment]
@anantcodes I have fix all that, do check!

File: lib/main.dart [hunk 4]
```diff
@@ -1,17 +1,21 @@
...
 import 'login.dart';
 import 'splash.dart';
 
 void main() {
-  runApp(
-    MaterialApp(
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
```
Comment (anantcodes) [inline lib/main.dart:10]: I don't think it's important to change the main.dart file.
Please do not modify the same.

**Review by anantcodes — CHANGES_REQUESTED** [review]

**Comment by anantcodes** [comment]
Please do the required changes and squash the commits.
Thanks!

File: lib/main.dart (hunk 4 above)
Comment (devkishor8007) [reply lib/main.dart:10]: It's not important but in the test>widget_test.dart, there is an error so I have done this and Should I change this?

Comment (devkishor8007) [reply lib/main.dart]: I have change this also do check and Is it ok?

**Pull request merged**